x/aswap/handler.go

```diff
@@ -9,13 +9,21 @@ import (
 	"github.com/iov-one/weave/migration"
 	"github.com/iov-one/weave/x"
 	"github.com/iov-one/weave/x/cash"
+	"github.com/tendermint/tendermint/libs/common"
 )
 
 const (
 	// pay swap cost up-front
 	createSwapCost  int64 = 300
 	returnSwapCost  int64 = 0
 	releaseSwapCost int64 = 0
+
+	// currently set to two days
+	dayInSeconds   = 86400
```

Comment:
To avoid putting `seconds` in the name, you can write it as
```
day = 24 * time.Hour
somethingElse = 2 * day  / time.Second // measured in seconds
```

I think even better is to declare it as duration and divide by `time.Seconds` where it must be represented as seconds. This way you maintain the helpful type that helps with time granularity.

Reply:
Yeah, I just figured out that UnixTime type has the ability to `.Add` actual durations, so I have converted everything to that.  Good catch